driver/esp32/ili9XXX.py

```diff
@@ -107,7 +111,7 @@ def __init__(self,
         miso=5, mosi=18, clk=19, cs=13, dc=12, rst=4, power=14, backlight=15, backlight_on=0, power_on=0,
         spihost=esp.HSPI_HOST, spimode=0, mhz=40, factor=4, hybrid=True, width=240, height=320, start_x=0, start_y=0,
         invert=False, double_buffer=True, half_duplex=True, display_type=0, asynchronous=False, initialize=True,
-        color_format=None
+        color_format=None, swap_rgb565_bytes=False
```

Comment:
color_format should not be defaulted to `None`. This is how LVGL now determines how the buffer data is to be stored. It should be defaulted to lv.COLOR_FORMAT.RGB888.

There needs to be a test that gets done so if a user inadvertiantly sets the `swap_rgb565_bytes` flag to true but has a color format other than RGB565 specified the flag will automatically get switched to  `False`

Reply:
> It should be defaulted to lv.COLOR_FORMAT.RGB888.

RGB565 is more common, so I'd choose it as default.

Reply:
Fixed